chainercv/transforms/bbox/non_maximum_suppression.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import division
+import numpy as np
+
+
+def non_maximum_suppression(bbox, threshold, limit=None, return_param=False):
+    """Suppress bounding boxes according to their Jaccard overlap.
+
+    This method checks each bounding box sequentially and selects the box
+    if the Jaccard overlap between the box and previously selected boxes
+    is less than :obj:`threshold`. This method is mainly used as postprocessing
+    of object detection. In this case, the bounding boxes should be sorted in
+    descending order.
```

Comment:
Nice.
How about this.

> the bounding boxes should be sorted by their confidence scores in descending order so that ones with more confidence get prioritized.